index.html

```diff
@@ -0,0 +1,108 @@
+            <div class="OFFF" >Granny gives everyone the finger, and other tips from OFFF Barcelona</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-XOzt5UX5xp2iXq.jpg" alt="">
+            <div class="helloWorld">Hello world, or, in other words, why this blog exists</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-wwIuzf7bbgI.jpg" alt="">
+            <div class="regarding">Here are some things you should know regarding how we work </div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-T3EhyoGpwJ.jpg" alt="">
+            <div  class="updatingList">Updating list of 50+ sources on distributed teams,
+                remote work, and how to make it all work better</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-aZU0MMuMOA5pq.jpg" alt="">
+            <div class="fewWordd">A few words about this blog platform, Ghost, and how this
+                site was made</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-X8Lnt3rhtf0Q.jpg" alt="">
+            <div class="modern" >How modern remote working tools get along with Old
+                School Cowboy's methods
+            </div>
+        </div>
+        <div class="blackSquare">
+            <div class="squareFirstText">
+
+                <div class="textCenter" class="DIGITAL" class="boldText">DIGITAL PRODUCT DESIGN&nbsp;</div>
+                <div class="textCenter" class="work">REMOTE WORK&nbsp;</div>
+                <div class="textCenter" class="US" class="boldText">UX DESIGN&nbsp;</div>
+                <div class="textCenter" class="teams">DISTRIBUTED TEAMS&nbsp;</div>
+                <div class="textCenter" class="creativity" class="boldText">CREATIVITY&nbsp;</div>
+                <div class="textCenter" class="strategy">STRATEGY&nbsp;</div>
+                <div class="textCenter" class="suspense" class="boldText">SUSPENSE&nbsp;</div>
+                <div class="textCenter" class="growth">GROWTH&nbsp;</div>
+            </div>
+
+            <div class="informationText">
+                <div class="bottomRose"><img width="245px" height="35px"
+                        src="NORDIC ROSE.png" alt="">
+                </div>
+
+                <div class="lorem">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis eu velit tempus erat
+                    egestas efficitur. In hac habitasse platea dictumst. Fusce a nunc eget ligula.</div>
+                <div class="links">
+                    <div class="oneLink">Twitter</div>
```

Comment:
1. why do you use `div` for links? it should be `a`. again, read about semantics and use semantic tags
2. I think all these links share the same style, so use the same class. review your page again and think about how you can reduce number of classes